src/RSocketServer.h

```diff
@@ -85,6 +85,12 @@ class RSocketServer {
 
   void shutdownAndWait();
 
+  /**
+   * Gets the port the ConnectionAcceptor is listening on.  Returns folly::none
+   * if this server is not listening on a port.
+   */
+  folly::Optional<uint16_t> listeningPort() const;
```

Comment:
Please let's not add this method to ConnectionAcceptor or RSocketServer. It looks like TCP specific method which would fit better to TcpConnectionAcceptor. 
For HTTP, Thrift and other servers, the method would mean nothing.